packages/ListBoxBrowser/src/ListBoxBrowser.styles.js

```diff
@@ -1,19 +1,31 @@
-import { css } from "styled-components";
+import styled, { css } from "styled-components";
 import tokens from "@paprika/tokens/lib/tokens";
 
-export const flex = css`
+export const Flex = styled.div`
   align-items: flex-start;
   display: flex;
   justify-content: center;
 
   [data-pka-anchor="listbox-content-inline"]:last-child {
     border-left: 1px solid ${tokens.border.color};
   }
+
+  ${({ isVisibleRoot }) => {
+    return isVisibleRoot
+      ? ""
+      : css`
+          display: inline;
+          [data-pka-anchor="listbox-content-inline"]:last-child {
+            border: 0;
+            border-radius: ${tokens.border.radius};
+          }
+        `;
```

Comment:
Can we define this above as `multiColumnStyles` (or something like that) and then collapse line 13 to a single, more readable line?

Reply:
to me is more understandable this way I can see actually what it's happening instead of having to go up on abstraction, and understand what multiColumStyles means, I think this comes to preference, I'm not a real fan of small abstraction or condense this type of logic. 

to me this way you pretty much are seeing what it's happening and what it's being modifying this style. but again its preference.